inference/environments/minimal-ubuntu20.04-py38-cpu-inference/context/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM mcr.microsoft.com/azureml/inference-base-2004:{{latest-image-tag}}
+
+RUN conda create -n minimal \
+    python=3.8.13 pip=22.1.2
```

Comment:
You're missing the -p option for `conda create`. See https://github.com/Azure/azureml-assets/blob/main/training/general/src/environments/pytorch-1.11-ubuntu20.04-py38-cuda11-gpu/context/Dockerfile

Reply:
Do we need a -p option if we are specifying the -n (name) option. For the curated environment in AzuremlCli repo we followed this approach to avoid changing to root user.